### ISSUE TITLE
SoS must be able to set risk limit before Counties upload data

### ISSUE THREAD

**Issue by sfsinger19103** [FreeAndFair/ColoradoRLA#552]
![unabletosetrl](https://user-images.githubusercontent.com/14353822/29851299-330df2dc-8ce8-11e7-99c0-22ac0494b8ea.png)

Rule 25 says that the risk limit must be set before the election, and counties can't upload files until after the election. See screenshot.

**Comment by sfsinger19103** [comment]
Or maybe there was another reason that system was unable to set risk limit. Error message didn't actually say it was waiting on Counties.

**Comment by ranweiler** [comment]
I believe this is only a backend ASM constraint right now. @dmzimmerman?

**Comment by dmzimmerman** [comment]
Um... the DoS can set risk limits whenever they want, right now. I did it just a little while ago.

**Comment by dmzimmerman** [comment]
As in, I started the client, signed in as the DoS, and the very first thing I did before even logging in as a county, on a fresh database, was set a risk limit.

**Comment by dmzimmerman** [comment]
Unless you can actually replicate this as an issue, I claim it's not one.

**Comment by ranweiler** [comment]
Okay, this is definitely closed, at least as of 64eade7143166de694b49b579c2f8191addedb2a. Something else may prevented the risk limit from being set. If we can repro a legitimate risk limit being unable to be set, then we can open a new issue for that.
